.github/workflows/version-change.yml

```diff
@@ -0,0 +1,56 @@
+name: Version Change
+on:
+  workflow_dispatch:
+    inputs:
+      option:
+        description: "Option"
+        required: true
+        default: 'append'
+        type: choice
+        options:
+          - append
+          - prepend
+          - remove
+      version:
+        description: "Version"
+        required: true
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - name: Remove Version
+        if: ${{ inputs.option == 'remove' }}
+        run: |
+          cat supported_cli_versions.json | jq 'del(.[] | select(. == "${{ inputs.version }}"))' > supported_cli_versions_temp.json
+      - name: Append Version
+        if: ${{ inputs.option == 'append' }}
+        run: |
+          cat supported_cli_versions.json | jq '. += ["${{ inputs.version }}"]' > supported_cli_versions_temp.json
```

Comment:
Are there situations where a maintainer will want to remove or append a version? In the rare case of there was a mistake, it's fine to manually edit the file. 

I'm suggesting this since I think this workflow file can be made simpler by removing the input and always prepending.

Reply:
Cool, I can remove the other options - I thought it might be easier to just add or remove versions whenever we needed 😄